config/schedule.rb

```diff
@@ -0,0 +1,21 @@
+every 1.day, :at => '4:30 am' do
+  runner "Passenger.update_active"
```

Comment:
The `update_active` method you defined is a model method, not a class method, so this definitely won't work.

Also, I think you can just write `every :day`.

Also it's nice to remove all of those comments ⬇️  once you're actually using a file for its intended purpose.